src/library/Uno.Material/Styles/Controls/v2/PasswordBox.xaml

```diff
@@ -471,32 +471,26 @@
 		<Setter Property="Template">
 			<Setter.Value>
 				<ControlTemplate TargetType="PasswordBox">
-					<Border x:Name="RootBorder"
-							BorderBrush="{TemplateBinding BorderBrush}"
-							BorderThickness="{TemplateBinding BorderThickness}"
-							CornerRadius="{TemplateBinding CornerRadius}"
-							Padding="{ThemeResource OutlinedPasswordBoxBorderPadding}">
```

Comment:
resources whose references are all removed, should also be removed
this change also need to be reflected, doc and c#markup